packages/manager/modules/bm-server-components/src/tasks/tasks.class.js

```diff
@@ -0,0 +1,31 @@
+import { TASK_STATUS } from './constants';
+
+export default class DedicatedServerTask {
+  constructor(lastUpdate, comment, action, status) {
+    Object.assign(this, {
+      lastUpdate,
+      comment,
+      action,
+      status,
+    });
+  }
+
+  isReady() {
+    return this.status === TASK_STATUS.DONE;
+  }
+
+  isUpdating() {
+    return (
+      this.status === TASK_STATUS.DOING ||
+      this.status === TASK_STATUS.INIT ||
+      this.status === TASK_STATUS.TODO
```

Comment:
`[TASK_STATUS.DOING, TASK_STATUS.INIT, TASK_STATUS.TODO].includes(this.status)`

Reply:
This looks simple but takes more time. Create a array and than loop through the array to check for the element.